### ISSUE TITLE
How do I set layer-wise LR multiplier?

### ISSUE THREAD

**Issue by KovenYu** [fchollet/keras#7840]
Hello!

I've tried for some whiles to figure out how to set different learning rate for some specific layers. E.g., when I'm using a CNN (say, ResNet), I would like to set the LR of BN layers (or more specially, the parameter BETA of BN layers) to (0.1 * global LR). But it turns out that I cannot simply achieve this without modifying lots of the database.

So, could anyone tell me how to set layer-wise LR multiplier, or some other ways to do the same work?

Thanks in advance!

**Comment by Zerphed** [comment]
Bump. I would be very interested in any easy way of achieving this for transfer learning purposes as well.

**Comment by kgrm** [comment]
The simplest was I see of accomplishing this is to implement a custom optimizer object that wraps an existing one but scales some of the incoming gradients.

**Comment by KovenYu** [comment]
# @kgrm I will try it. Thx!

**Comment by KovenYu** [comment]
@kgrm
